src/metrics/master.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "metrics/metrics.h"
+
+#ifdef HAVE_PROMETHEUS
+#include <prometheus/counter.h>
+#include <prometheus/registry.h>
+
+namespace metrics::master {
+
+struct Master {
```

Comment:
Why use `struct` instead of `class`?

Reply:
struct is public by default compared to class, which I generally prefer if I have few/no members that are private and no inheritance is intended for it (i.e it's a simple data structure).